curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5f07be6ef7412fbad0c5626b.md

```diff
@@ -9,6 +9,15 @@ dashedName: step-16
 
 Before adding any new content, you should make use of a `section` element to separate the cat photos content from the future content.
 
+The `section` element is used to define sections in a document, such as chapters, headers, footers, or any other sections of the document. It is a semantic element that helps with SEO and accessibility.
+
+```html
```

Comment:
this is another one that comes up a lot on the forum
https://forum.freecodecamp.org/search?q=step%2016%20%20%23html-css%20cat%20photo%20app%20after%3A2024-03-01

I think the issue is that the concept of nesting is still to new and campers are making that connection yet on how to do that on their own without an example